app/build.gradle

```diff
@@ -491,4 +494,43 @@ afterEvaluate {
             }
         }
     }
+
+    // Format test output. Ported from AC #2401
+    tasks.matching {it instanceof Test}.all {
+        testLogging.events = []
+
+        def out = services.get(StyledTextOutputFactory).create("an-ouput")
+
+        beforeSuite { descriptor ->
+            if (descriptor.getClassName() != null) {
```

Comment:
When would `descriptor.getClassName() == null`? Is it possible to have tests outside of a suite?

Reply:
I do not remember the situation, but it did happen in AC.